mysql/mysqldump.go

```diff
@@ -238,13 +246,35 @@ func toSchemaKeys(columns []*ast.IndexPartSpecification) (keys []schema.Key) {
 	return keys
 }
 
+// toForeignKeys converts a string list of MySQL foreign keys to
+// schema foreign keys.
+func toForeignKeys(conv *internal.Conv, fk *ast.Constraint) (fkey schema.ForeignKey) {
+	columns := fk.Keys
+	referTable, err := getTableName(fk.Refer.Table)
+	if err != nil {
+		conv.Unexpected(err.Error())
+		return schema.ForeignKey{}
+	}
+	referColumns := fk.Refer.IndexPartSpecifications
+	var colNames, referColNames []string
+	for i, column := range columns {
+		colNames = append(colNames, column.Column.Name.String())
+		referColNames = append(referColNames, referColumns[i].Column.Name.String())
+	}
+	fkey = schema.ForeignKey{Columns: colNames,
```

Comment:
Suggest breaking line at "Column" so that all of the fields of the struct line up.